src/str.js

```diff
@@ -1364,7 +1364,7 @@ Sk.builtin.str.prototype.nb$remainder = function (rhs) {
         }
     };
     ret = this.$jsstr().replace(regex, replFunc);
-    if (index < Sk.builtin.asnum$(Sk.builtin.len(rhs))) {
+    if (rhs instanceof Sk.builtin.tuple && index < Sk.builtin.asnum$(Sk.builtin.len(rhs))) {
```

Comment:
I think since we know this is a tuple we can use `rhs.sq$length()` which will return a js number.

Reply:
Oh, yes! I for get it :(